test/integration_test.go

```diff
@@ -141,15 +144,30 @@ func (s *MySuite) TestIntegration(c *check.C) {
 	client.Get(hostsRelURL, &hostList)
 	myLog(c, "Host list: ", hostList)
 	c.Assert(len(hostList), check.Equals, 0)
-	newHostReq := common.HostMessage{Ip: "10.10.10.10", RomanaIp: "10.0.0.1/16", AgentPort: 9999, Name: "HOST1000"}
 
+	// Find some romana IPs that we can use... Because the agent checks for those
+	addrs, err := net.InterfaceAddrs()
+	if err != nil {
+		c.Error(err)
+	}
+
+	possibleRomanaIps := make([]string, 0)
+	for _, addr := range addrs {
+		strAddr := addr.String()
+		if strings.ContainsAny(strAddr, ":") {
```

Comment:
What does the presence of a ':' signify and why do we ignore that one? Maybe a little comment to explain that.